### PR TITLE
Activity tab formatting and scroll

### LINKED ISSUE #4709: Activity tab formatting and scroll
### Scroll
The top section with event selection should stay fixed and not scroll. 
image
- only the data should scroll

### Formatting
on mobile its not clear what row of date you are looking at 
Screenshot_2023-01-15-12-19-08-149_com android chrome

should maybe look more like this:
image

- only adding divider
- making the event name non-caps
- the tags (event, price, from, to etc.) should be 12px small text in k-grey color 
- please make the section with filters fit the container, maybe jumping it to the next row.

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [x] Redesign
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4709
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by exezbcz** [comment]
can you please make the selection stretch so its not all left
something like here: 
image



image

the divider should not be up to edges and it should have shade color
- please add some margin as well 


image
- offers tab has now top margin


image
- can we make also this text noncaps, only with first letter capital.

**Comment by roiLeo** [comment]
>     * please add some margin as well
> 
> 
> image

What do you think about removing the "USD Price" column? we can try to integrate it directly with the price 
__ex: 0.1 KSM ($3)__

**Comment by yangwao** [comment]
> What do you think about removing the "USD Price" column? we can try to integrate it directly with the price
> **ex: 0.1 KSM ($3)**

we've been through this thought process w/ @exezbcz in October and we steer away as if price will be different it will loose table feeling and everything looking nice. 

> image
> 

I think offers should use mobile table to not create second row?

File: components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue [hunk 1]
```diff
@@ -120,10 +120,16 @@ const expirationTime = (block) => {
...
-    return $i18n.t('offer.withdrawn')
+  switch (status) {
+    case OfferStatusType.WITHDRAWN:
+      return $i18n.t('offer.withdrawn')
+    case OfferStatusType.ACTIVE:
+      return $i18n.t('offer.active')
+    case OfferStatusType.ACCEPTED:
+      return $i18n.t('offer.accepted')
+    default:
+      return status
```
Comment (roiLeo) [inline components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue:131]: this might break #4739 ?

Comment (Jarsen136) [reply components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue:131]: that's OK, a small code conflict

**Comment by Jarsen136** [comment]
> can you please make the selection stretch so its not all left something like here: image
> 
> image
> 
> the divider should not be up to edges and it should have shade color
> 
> * please add some margin as well
> 
> image
> 
> * offers tab has now top margin
> 
> image
> 
> * can we make also this text noncaps, only with first letter capital.

✅ All fixed

**Comment by Jarsen136** [comment]
> I think offers should use mobile table to not create second row?

I tried to achieve it without success : )
I have tried to set this config to the o-table component (oruga), but it seem does not work. Do you have any ideas? @roiLeo @preschian 

image
docs: https://oruga.io/components/table.html#props-1

**Comment by roiLeo** [comment]
> https://oruga.io/components/table.html#props-1

have you tried to set `mobile-cards` in `<o-table>` element?
works for me in oruga doc website on responsive view

**Comment by Jarsen136** [comment]
> > https://oruga.io/components/table.html#props-1
> 
> have you tried to set `mobile-cards` in `<o-table>` element? works for me in oruga doc website on responsive view
> 
> edit: nvm, `true` by default and it's already set up
> 

Thanks. In other words, seems that we have no way to force the table to display as mobile card mode, when in pc screen.

**Comment by roiLeo** [comment]
> Thanks. In other words, seems that we have no way to force the table to display as mobile card mode, when in pc screen.

We can still iterate this with a custom list component

**Comment by exezbcz** [comment]
nice! i have still few issues
image
- can you please add more space, we need more seperation so its clear at what data you are looking at

not sure if you can reproduce it but cards are overflowing the screen.
image
from something aroud 1215 its not responsive.

**Comment by Jarsen136** [comment]
> nice! i have still few issues 
> 
> * can you please add more space, we need more seperation so its clear at what data you are looking at
> 
> not sure if you can reproduce it but cards are overflowing the screen. 

✅  Fixed

**Comment by exezbcz** [comment]
> > nice! i have still few issues
> > 
> > * can you please add more space, we need more seperation so its clear at what data you are looking at
> > 
> > not sure if you can reproduce it but cards are overflowing the screen.
> 
> ✅ Fixed
> 
> cc @exezbcz

aaaand the gap here? 
image
between the tabs and column names


and also in case of horizontal scroll, can we make the scroll visible so its clear that there is a content? 
image


otherwise alright! Thanks 🚀

**Comment by Jarsen136** [comment]
> aaaand the gap here?
> between the tabs and column names
> and also in case of horizontal scroll, can we make the scroll visible so its clear that there is a content?

✅  done

Is the gap properly now?
image

How about making the scrollbar visible while scrolling?

Kapture 2023-01-19 at 01 32 41

**Comment by exezbcz** [comment]
> > aaaand the gap here?
> > between the tabs and column names
> > and also in case of horizontal scroll, can we make the scroll visible so its clear that there is a content?
> 
> ✅ done
> 
> Is the gap properly now? image
> 
> How about making the scrollbar visible while scrolling?
> 
> Kapture 2023-01-19 at 01 32 41 [ Kapture 2023-01-19 at 01 32 41 ](https://user-images.githubusercontent.com/31397967/213253286-32ed72c9-555d-41de-85c9-7ea6575aaee0.gif) [ ](https://user-images.githubusercontent.com/31397967/213253286-32ed72c9-555d-41de-85c9-7ea6575aaee0.gif)

afaik it was even visible before

**Comment by Jarsen136** [comment]
> afaik it was even visible before

I'm not sure what you mean. They are in different screen sizes. And the scrollbar would only display in small screen size. 
As for the widescreen, the offer table could display fully without  scrollbar
image

**Comment by exezbcz** [comment]
> > afaik it was even visible before
> 
> I'm not sure what you mean. They are in different screen sizes. And the scrollbar would only display in small screen size. As for the widescreen, the offer table could display fully without scrollbar image

image
now i have two scrollbars, even if i dont need them to move something 

this looks good:
image
- is there any way to leave the bottom scrollbar for horizontal scroll visible when the whole tab does not fit and has to be smaller?

**Comment by Jarsen136** [comment]
> image now i have two scrollbars, even if i dont need them to move something
> 

Sorry, I could not reproduce the double scrollbar in my mac computer. It should be related to the windows os issue.
Kapture 2023-01-20 at 01 20 18


> 
> * is there any way to leave the bottom scrollbar for horizontal scroll visible when the whole tab does not fit and has to be smaller?

I think yes. How about creating another issue for both two problems above? They are both related to the scrollbar behaviour.

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue [hunk 2]
```diff
@@ -130,3 +130,8 @@ const formatPrice = (price) => {
   return formatBalance(price, decimals.value, tokenSymbol)
 }
 </script>
+<style lang="scss" scoped>
+.gallery-item-activity-table {
+  overflow-y: scroll;
```
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue]: Thanks. Updated it.

**Comment by exezbcz** [comment]
> 

will add to gallery item v1.1, thanks!

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 35.91 USD/KSM ~ 1.392 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xe9b9cbc5a370d20e3e928c2e6d3539115b69a7fb8c2915fbba67f66d9942a745](https://kusama.subscan.io/extrinsic/0xe9b9cbc5a370d20e3e928c2e6d3539115b69a7fb8c2915fbba67f66d9942a745)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**